content/c-sharp/concepts/Conditionals/conditionals.md

```diff
@@ -0,0 +1,152 @@
+---
+Title: 'conditionals'
+Description: 'Compares inputs and gives an boolean output depending on it being True or False.'
+Subjects:
+  - 'Computer Science'
+  - 'Web Dev'
+  - 'App Dev'
+  - 'Game Dev'
+Tags:
+  -
+CatalogContent:
```

Comment:
Please refer to the [catalog content](https://github.com/Codecademy/docs/blob/main/documentation/catalog-content.md) document.